src/Modal.js

```diff
@@ -466,4 +485,7 @@ Modal.Header = Header;
 Modal.Title = Title;
 Modal.Footer = Footer;
 
+Modal.TRANSITION_DURATION = 300;
+Modal.BACKDROP_TRANSITION_DURATION = 150;
```

Comment:
From what I understand, this *is* configurable. It's a static property on `Modal`, so you only have to change it directly, i.e.

```js
require("react-bootstrap/lib/Modal").TRANSITION_DURATION = 1234;
```

Anywhere in your code, to modify the default value. Did you mean that it should be configurable per-instance? That doesn't sound very consistent to me.